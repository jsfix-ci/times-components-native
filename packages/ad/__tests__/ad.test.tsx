import React from "react";
import { render, screen } from "@testing-library/react-native";
import "react-native-device-info";
import "./mocks";

import Ad from "@times-components-native/ad";

jest.mock("react-native-device-info", () => {
  return {
    isTablet: jest.fn(),
  };
});

test("** AD **", async () => {
  beforeEach(() => {
    jest.resetAllMocks();
  });

  render(
    <Ad
      adConfig={{ sectionName: "News", slug: "slug", isLive: false }}
      slotName="ad-section"
    />,
  );

  expect(screen.toJSON()).toMatchSnapshot();
});
