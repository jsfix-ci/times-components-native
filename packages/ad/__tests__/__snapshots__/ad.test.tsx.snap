// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`** AD ** 1`] = `
<View
  inViewport={false}
  onViewportEnter={[Function]}
  style={
    Object {
      "height": 0,
      "width": 750,
    }
  }
/>
`;
